### DISCUSSION TITLE
OEM Information

### DISCUSSION THREAD

**Discussion by X9VoiD** [OpenTabletDriver/OpenTabletDriver#534] [General]
To improve OpenTabletDriver.Installer, we have to collect some information about OEM drivers. 

# Steps

1. Run powershell, then type `pnputil -e`, save output to a text file then upload here.

2. From the same powershell window, type
```ps
Get-ItemProperty HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall\*, HKLM:\Software\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\* | Select-Object DisplayName, UninstallString | Sort-Object DisplayName
```
Find the line where it looks like the tablet driver then paste the DisplayName to `Driver Name` using the sample format provided below.

# OEM Brand / Tablet

XP-Pen G640S

## Driver Name

Pentablet version 3.0.6.201126

> The `version xx.yy.zz` is not required, it's only there since `DisplayName` just so happened that it includes it

## PnPUtil Output

[pnputil-output.txt](https://github.com/InfinityGhost/OpenTabletDriver/files/5667177/pnputil-output.txt)

**Comment by Mrcubix** [comment]
# OEM Brand / Tablet
Gaomon S620

## Driver Name
Gaomon Tablet v14.8.133.1259
GaomonTablet                                              > **HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall**
Huion Tablet v14.8.137.1273
VKTabletDriver

## PnPUtil Output
[log.txt](https://github.com/InfinityGhost/OpenTabletDriver/files/5667407/log.txt)
https://cdn.discordapp.com/attachments/628651971267788800/788449509571166299/log.txt

**Reply by X9VoiD** [reply]
Format is now updated, please update the contents of this one.

**Comment by Joromonni** [comment]
<h1>OEM Brand / Tablet</h1>

<p>Wacom DTK-1300 (Cintiq 13HD)</p>

<h2>Driver Name</h2>

<p>Wacom Tablet v1.0.2.3<sub>i think</sub></p>

<h2>PnPUtil Output</h2>

[log.txt](https://github.com/InfinityGhost/OpenTabletDriver/files/5671234/log.txt)

**Reply by X9VoiD** [reply]
Is the uninstaller there the thing that uninstalls the wacom drivers (are the processes you listed gone after running that)?

**Reply by X9VoiD** [reply]
Format is now updated, please update the contents of this one.

**Reply by Joromonni** [reply]
updated but heres a pic of what the DisplayName and UninstallString look like together 
![powershell_HPkvXpEMwY](https://user-images.githubusercontent.com/40163855/101916975-55950d00-3bd0-11eb-98bf-b5f375b4471b.png)

**Comment by second2050** [comment]
# OEM Brand / Tablett
Wacom CTL-490 (Intuos Draw)

# Driver Name
Wacom Tablett

Note: no version string was displayed.

# PnPUtil Output
[PNPUTIL.LOG](https://github.com/InfinityGhost/OpenTabletDriver/files/5689281/PNPUTIL.LOG)

**Comment by enoslayd** [comment]
### **OEM Brand / Tablet**
XP-Pen G430s

### **Driver Name**
Pentablet 3.0.6.201126 (the beta new ui version)

### **PnPUtil Output**
[pnputil.txt](https://github.com/InfinityGhost/OpenTabletDriver/files/5727804/pnputil.txt)

*since the powershell command doesnt work for me for some reason, i tried to find it manually in registry and screenshotted it*
![Untitled](https://user-images.githubusercontent.com/41866328/102849611-7708bb00-4452-11eb-84a1-c3f790387fed.png)

**Comment by jamesbt365** [comment]
We no longer use the installer and we have TabletDriverCleanup, closing.
